### ISSUE TITLE
vs2022  is supported cuda?

### ISSUE THREAD

**Issue by ganlubbq** [codecryptanalysis/mccl#14]
hi，I have successfully compiled this program in VS 2022. I want to know in VS 2022  the Mccl utilizes CUDA resources or not？

**Comment by cr-marcstevens** [comment]
No, there is currently no real part of mccl that actually uses cuda yet. So both for windows and linux.
I added it to the setup due to ongoing work.

**Comment by ganlubbq** [comment]
thanks your reply
